README.md

```diff
@@ -41,6 +41,10 @@ cd arc-mlir/; ./arc-to-mlir-build; cd -
 
 # Run tests
 cd arc-mlir/build/llvm-build/; ninja check-arc-mlir; cd -
+
+# Compile and run arc-script
+cd arc-script/
+cargo run --release --features=repl -- --check repl
```

Comment:
```
$ cargo run --release --features=repl -- --check repl
error: --features is not allowed in the root of a virtual workspace
note: while this was previously accepted, it didn't actually do anything
```

Removing `--features=repl` gives:
```
 $ cargo run --release -- --check repl
    Finished release [optimized] target(s) in 0.09s
     Running `target/release/arc-script --check repl`
error: Found argument 'repl' which wasn't expected, or isn't valid in this context

If you tried to supply `repl` as a PATTERN use `-- repl`

USAGE:
    arc-script [FLAGS] [OPTIONS] <SUBCOMMAND>

For more information try --help
frej@emu .../arc/arc-script $
```

This is with cargo and rustc 1.46.0-nightly.

Reply:
Ah forgot about the workspace.

Reply:
Or did you run it from the current commit? It is strange that you got the first message, there should not be a workspace on this branch.

Reply:
I merged arc-script into mlir, and ran it there.

Reply:
Ok, now it should work. I can squash the commits if we merge.

Reply:
Those instructions work, please merge the two commits. If things are different between the mlir and the arc-script branches, move this change to the arc-script branch.